common/fe-ste-models.go

```diff
@@ -110,7 +111,7 @@ type PartNumber uint32
 type Version uint32
 type Status uint32
 
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
can these be reverted?

Reply:
*the comment changes specifically

Reply:
this formatting setting can be changed in GoLand.
File -> Settings -> Editor -> Code Style -> Go -> Other -> Add a leading space to comments should be unchecked.